app/views/event_name.erb

```diff
@@ -0,0 +1,51 @@
+<div id="sources">
+  <div class='col-md-12 navbar navbar-default navbar-static-top navbar-inverse'>
+    <a href='/' class='navbar-brand'>Rush Hour</a>
+    <ul class='nav navbar-nav navbar-right'>
+      <li><a href = "https://www.youtube.com/watch?v=ZFq_TFyhzmo">Full Movie</a></li>
+      <li><a href = "https://www.redbubble.com/shop/jackie+chan">Store</a></li>
+      <li class='dropwdown'>
+        <a href = '#' class='dropdown-toggle' data-toggle='dropdown'>Events
+        <span class="caret"></span></a>
+        <ul class='dropdown-menu'>
+          <% @events.each do |event| %>
+            <li><a href="/sources/<%= @client.identifier %>/events/<%= event %>"><%= event %></a></li>
+          <% end %>
+        </ul>
+      </li>
+    </ul>
+    <form class="navbar-form" role="search" action="/redirect" method="get">
+			<div class="input-group">
```

Comment:
also - watch your spacing here. follow Ruby conventions - a tab is two spaces. i've heard of interviewers rejecting candidates because their spacing did not follow the convention